index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h2>John Doe</h2>
+    <!-- <i class="material-icons">menu</i> -->
+    <img class="hover-button" src="/Icon-Menu.png" alt="">
+  </header>
+
+  <section>
+    <img src="Top-circles.png" alt="">
```

Comment:
- [ ] This section should have the proper background image (decorations should be with background images, not `img` tags). As Background image helps to write the text over them as well.